src/Foundation/NSMetadataItem.cs

```diff
@@ -177,5 +187,1014 @@ bool GetBool (NSString key)
+		public double FocalLength {
+			get {
+				return GetDouble (NSMetadataQuery.FocalLengthKey);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public string AcquisitionMake {
+			get {
+				return (string) Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.AcquisitionMakeKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public string AcquisitionModel {
+			get {
+				return (string) Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.AcquisitionModelKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public double IsoSpeed {
+			get {
+				return GetDouble (NSMetadataQuery.IsoSpeedKey);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public nint Orientation {
+			get {
+				return GetNInt (NSMetadataQuery.OrientationKey);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public string [] LayerNames {
+			get {
+				return NSArray.StringArrayFromHandle (GetHandle (NSMetadataQuery.LayerNamesKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public double WhiteBalance {
+			get {
+				return GetDouble (NSMetadataQuery.WhiteBalanceKey);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public double Aperture {
+			get {
+				return GetDouble (NSMetadataQuery.ApertureKey);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public string ProfileName {
+			get {
+				return (string) Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.ProfileNameKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public nint ResolutionWidthDpi {
+			get {
+				return GetNInt (NSMetadataQuery.ResolutionWidthDpiKey);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public nint ResolutionHeightDpi {
+			get {
+				return GetNInt (NSMetadataQuery.ResolutionHeightDpiKey);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public nint ExposureMode {
+			get {
+				return GetNInt (NSMetadataQuery.ExposureModeKey);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public double ExposureTimeSeconds {
+			get {
+				return GetDouble (NSMetadataQuery.ExposureTimeSecondsKey);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public string ExifVersion {
+			get {
+				return (string) Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.ExifVersionKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public string CameraOwner {
+			get {
+				return (string) Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.CameraOwnerKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public nint FocalLength35mmKey {
+			get {
+				return GetNInt (NSMetadataQuery.FocalLength35mmKey);
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public string LensModel {
+			get {
+				return (string) Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.LensModelKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public string ExifGpsVersion {
+			get {
+				return (string) Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.ExifGpsVersionKey));
```

Comment:
Same: Use `NSString.FromHandle (IntPtr)` (and there are several more below as well).